client/account-recovery/reset-password-confirm-form/index.jsx

```diff
@@ -60,6 +68,8 @@ class ResetPasswordConfirmForm extends Component {
 
 		const { newPassword } = this.state;
 
+		const capturePasswordInputRef = ( passwordInput ) => this.passwordInput = passwordInput;
```

Comment:
Also, can we make this a class method instead. Then we can use `this.capturePasswordInpurRef` below